src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -322,4 +397,37 @@ private function createOrder(
 
         return $order;
     }
+
+    /**
+     * @param $count
+     *
+     * @return CustomerInterface[]
+     */
+    private function generateCustomers($count)
+    {
+        $customers = [];
+
+        for ($i = 0; $i < $count; $i++) {
+            $customer = $this->customerFactory->createNew();
+            $customer->setEmail(sprintf('john%s@doe.com', uniqid()));
```

Comment:
It was $i, but then I can't easily run this method twice (in 2 steps for example), because I will get non unique e-mails: 2 x john1@doe.com.